shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1111,6 +1121,59 @@ - (void)forceTouchesCancelled:(NSSet*)touches {
   [self dispatchTouches:touches pointerDataChangeOverride:&cancel event:nullptr];
 }
 
+#pragma mark - Touch events rate correction
+
+- (void)createTouchRateCorrectionVSyncClientIfNeeded {
+  NSAssert(_touchRateCorrectionVSyncClient == nil,
+           @"_touchRateCorrectionVSyncClient should be nil when setup it");
+  double displayRefreshRate = [DisplayLinkManager displayRefreshRate];
+  if (displayRefreshRate <= 60) {
```

Comment:
Use a small epsilon for double comparison.

Reply:
Done^_^